assets/js/modules/idea-hub/datastore/idea-state.test.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * `modules/idea-hub` data store: idea-state tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { STORE_NAME } from './constants';
+import {
+	createTestRegistry,
+	unsubscribeFromAll,
+} from '../../../../../tests/js/utils';
+import { enabledFeatures } from '../../../features';
+
+describe( 'modules/idea-hub idea-state', () => {
+	let registry;
+
+	const ideaStateFixture = {
+		name: 'ideas/7612031899179595408',
+		saved: false,
+		dismissed: false,
+	};
+
+	beforeAll( () => {
+		API.setUsingCache( false );
+	} );
+
+	beforeEach( () => {
+		enabledFeatures.add( 'ideaHubModule' );
+		registry = createTestRegistry();
+	} );
+
+	afterEach( () => {
+		unsubscribeFromAll( registry );
+	} );
+
+	afterAll( () => {
+		API.setUsingCache( true );
+	} );
+
+	describe( 'actions', () => {
+		describe( 'updateIdeaState', () => {
+			it( "updates a given idea's state", async () => {
+				fetchMock.postOnce(
+					/^\/google-site-kit\/v1\/modules\/idea-hub\/data\/update-idea-state/,
+					ideaStateFixture
```

Comment:
I think this should be `{ body: ideaStateFixture }`, shouldn't it? If so, we will need to update other occurrences in this file too.